spec/Section 4 -- Introspection.md

```diff
@@ -151,6 +151,8 @@ type __Type {
   ofType: __Type
   # may be non-null for custom SCALAR, otherwise null.
   specifiedByURL: String
+  isDeprecated: Boolean!
```

Comment:
Hm. `__Type` includes non-objects (unions, interfaces, anything in `__TypeKind`). Maybe this field should be nullable like `possibleTypes`? Or defaults to `false` for any type that can't be deprecated?